app/views/candidate_mailer/ucas_match_initial_email_duplicate_applications.text.erb

```diff
@@ -0,0 +1,22 @@
+Dear <%= @application_form.full_name %>
+
+It looks like you submitted an application for <%= @course_name_and_code %> at <%= @provider_name %> on both DfE Apply for teacher training and UCAS Teacher Training Apply.
+
+This has been flagged to us, as well as to <%= @provider_name %>, as applying for the same course on both services is not allowed, as outlined by the terms of use.
+
+Action required – withdraw your duplicate application
+
+To ensure your application progresses effectively, you need to withdraw the course choice from either DfE Apply for teacher training or UCAS Teacher Training by <%= @date_to_withdraw_application_by %>.
+
+It’s up to you which system you choose to progress the choice on. If you have additional UCAS Teacher Training choices, you may want to keep the choice on UCAS Teacher Training, so all your choices are in one place.
+
+What happens if you don’t do anything?
+
+If you don’t withdraw your duplicate choice by <%= @date_to_withdraw_application_by %> (in 10 working days), we will automatically withdraw the choice removed from your UCAS Teacher Training application – meaning you’ll use DfE Apply for teacher training to progress this choice.
```

Comment:
`we will automatically withdraw the choice removed from your UCAS Teacher Training application`
We may need some content work on this 😬

Reply:
@AgaDufrat 🤣